icons/icon.svg

```diff
@@ -1 +1 @@
-<svg role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><title>ICON icon</title><path d="M21.605.021a2.397 2.397 0 00-.47 4.747 2.397 2.397 0 10.47-4.746zm-9.584 2.395C6.73 2.416 2.437 6.707 2.437 12c0 2.142.703 4.121 1.891 5.717l2.586-2.586a5.99 5.99 0 018.236-8.238l2.586-2.586a9.537 9.537 0 00-5.715-1.891zm7.692 3.867L17.127 8.87a5.989 5.989 0 01-8.236 8.238l-2.586 2.584a9.543 9.543 0 005.716 1.893c5.293 0 9.583-4.29 9.583-9.584 0-2.142-.704-4.12-1.891-5.717zM2.396 19.187a2.398 2.398 0 00-.47 4.749 2.402 2.402 0 002.822-2.823 2.399 2.399 0 00-2.352-1.925z"/></svg>
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>ICON icon</title><path d="M21.59664.0241a2.39612 2.39612 0 00-.46983 4.74526A2.39612 2.39612 0 1021.59665.0251zm-9.58048 2.39412c-5.28906 0-9.5805 4.28943-9.5805 9.58049 0 2.14121.70275 4.11949 1.89032 5.7149l2.58505-2.58505A5.9878 5.9878 0 0115.144 6.89358l2.58505-2.58505a9.5335 9.5335 0 00-5.7129-1.89031zm7.68918 3.86558l-2.58506 2.58605a5.9868 5.9868 0 01-8.23298 8.23498L6.30225 19.6879a9.5395 9.5395 0 005.7139 1.8923c5.29106 0 9.5795-4.28842 9.5795-9.58048 0-2.14122-.70375-4.1185-1.89031-5.7149zM2.39468 19.18307a2.39712 2.39712 0 00-.46982 4.74726 2.40112 2.40112 0 002.82096-2.82196 2.39812 2.39812 0 00-2.35114-1.9243z"/></svg>
```

Comment:
Looks like this path is at 5 decimals. Any chance you could reduce this?

Reply:
It worked after rebuilding it from the source https://icon.foundation/resources/image/icon_logo.svg 💪🏼 

Which was actually as well necesarry, as the icon slightly differed from the source.